lib/senders/templates/_versions.json

```diff
@@ -1,6 +1,6 @@
 {
   "lowRecoveryCodesEmail": 2,
-  "newDeviceLoginEmail": 1,
+  "newDeviceLoginEmail": 5,
```

Comment:
what was the logic here, v5 vs v2?

Reply:
Maybe some scripting strangeness? I'll update.

Reply:
I had that question too.